### ISSUE TITLE
Nested Fragments fail

### ISSUE THREAD

**Issue by caedmon** [speee/jsx-slack#53]
Attempting to nest Fragments with multiple children leads to exception:

```
TypeError: Cannot read property 'children' of undefined
     at JSXSlack (.../node_modules/@speee-js/jsx-slack/lib/jsx.js:23:60)
     at children.reduce (.../node_modules/@speee-js/jsx-slack/lib/jsx.js:27:69)
     at Array.reduce (<anonymous>)
     at toArray (.../node_modules/@speee-js/jsx-slack/lib/jsx.js:25:53)
     at Object.JSXSlack (.../node_modules/@speee-js/jsx-slack/lib/jsx.js:37:20)
     at ...
```

To reproduce:

```JSX
JSXSlack(
  <Blocks>
    <Fragment>
      <Fragment>
        <Section>A</Section>
        <Section>B</Section>
      </Fragment>
      <Fragment>
        <Section>C</Section>
        <Section>D</Section>
      </Fragment>
    </Fragment>
    <Fragment>
      <Fragment>
        <Section>A</Section>
        <Section>B</Section>
      </Fragment>
      <Fragment>
        <Section>C</Section>
        <Section>D</Section>
      </Fragment>
    </Fragment>
  </Blocks>
)
```

**Comment by yhatt** [comment]
Thanks for your reporting. It seems to be an edge case in specific structure with fragments. I'll try to fix it.

**Comment by yhatt** [comment]
Fixed in v0.9.2.
